src/inngest/functions/send-slack-message.ts

```diff
@@ -59,6 +59,19 @@ export const sendSlackMessage = inngest.createFunction(
     } else if (messageType === 'feedback') {
       channel = process.env.SLACK_EGGHEAD_FEEDBACK_CHANNEL_WEBHOOK ?? ''
 
+      try {
+        await postFeedbackToSlack({
+          webhookUrl: channel,
+          username: 'Eggo',
+          text: message,
+          attachments,
+        })
+      } catch (error) {
+        console.error('Error sending slack message', error)
+      }
+    } else if (messageType === 'error') {
+      channel = process.env.SLACK_EGGHEAD_MONITORING_CHANNEL_WEBHOOK ?? ''
```

Comment:
setting up a webhook for a channel is very straight forward, figuring out bot permissions has been a maze of ambiguity. I figured with something like this error reporting we will only need this one channel anyway.